### PR TITLE
Add Hadoop container.

### PR DESCRIPTION
**Pull request by daithiocrualaoich**
076eb17a0e9

### CODE REVIEW

File: docs/hadoop.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+::
+
+    sudo rkt run \
+      --port=50030-tcp:50030 \
+      --port=50060-tcp:50060 \
+      --port=50070-tcp:50070 \
+      --port=50075-tcp:50075 \
+      hadoop
+
+The main HDFS web dashboard is on port 50070.
```
Comment (passy) [inline docs/hadoop.rst]: Woa. Didn't know HDFS had a web interface.

File: hadoop/hadoop.acbuild.sh [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+TMP_DIR="$(mktemp -d -p "$DIR" hadoop.XXXXXX)"
+pushd "$TMP_DIR" > /dev/null
+
+
+################################################################################
+# Download Base Image
+################################################################################
+
+wget http://cdimage.ubuntu.com/ubuntu-base/releases/17.04/release/ubuntu-base-17.04-base-amd64.tar.gz
```
Comment (passy) [inline hadoop/hadoop.acbuild.sh]: Okay, I know this comes super late. But why such a massive distro as base image? Seems like in many cases like this one it's not actually required.

Comment (daithiocrualaoich) [reply hadoop/hadoop.acbuild.sh]: Familiarity and large package repository. I'll put looking at Apline Linux and alternatives on the backlog.

File: docs/hadoop.rst (hunk 1 above)
Comment (daithiocrualaoich) [reply docs/hadoop.rst]: It's not amazing for browsing the filesystem. Better for log messages and status.

**Pull request merged**